Bionic lacks strverscmp as well.
diff -uNr ../procps-3.3.17/Makefile.in ./Makefile.in > Makefile.in.patch
--- ../procps-3.3.17/Makefile.in	2021-02-09 15:41:25.000000000 +0530
+++ ./Makefile.in	2021-04-20 15:35:51.385653341 +0530
@@ -139,7 +139,7 @@
 @WITH_NCURSES_TRUE@	watch.1 \
 @WITH_NCURSES_TRUE@	top/top.1
 
-@CYGWIN_TRUE@@WITH_NCURSES_TRUE@am__append_20 = lib/strverscmp.c
+@WITH_NCURSES_TRUE@am__append_20 = lib/strverscmp.c
 @CYGWIN_FALSE@@WITH_NCURSES_TRUE@am__append_21 = \
 @CYGWIN_FALSE@@WITH_NCURSES_TRUE@	slabtop.1
 
@@ -367,8 +367,8 @@
 tload_DEPENDENCIES = ./proc/libprocps.la $(am__DEPENDENCIES_2)
 am__top_top_SOURCES_DIST = top/top.h top/top.c top/top_nls.h \
 	top/top_nls.c lib/fileutils.c lib/strverscmp.c
-@CYGWIN_TRUE@@WITH_NCURSES_TRUE@am__objects_1 =  \
-@CYGWIN_TRUE@@WITH_NCURSES_TRUE@	lib/strverscmp.$(OBJEXT)
+@WITH_NCURSES_TRUE@am__objects_1 =  \
+@WITH_NCURSES_TRUE@	lib/strverscmp.$(OBJEXT)
 @WITH_NCURSES_TRUE@am_top_top_OBJECTS = top/top.$(OBJEXT) \
 @WITH_NCURSES_TRUE@	top/top_nls.$(OBJEXT) \
 @WITH_NCURSES_TRUE@	lib/fileutils.$(OBJEXT) $(am__objects_1)
