diff -uNr ../procps-3.3.17/pgrep.c ./pgrep.c > ../pgrep.c.patch
--- ../procps-3.3.17/pgrep.c	2021-02-09 15:41:25.000000000 +0530
+++ ./pgrep.c	2021-04-21 15:22:58.741546689 +0530
@@ -799,7 +799,9 @@
 		strcat (opts, "e");
 	} else
 #endif
-        if (strcmp (program_invocation_short_name, "pkill") == 0) {
+		// replace program_invocation_short_name with argv[0]
+       // if (strcmp (program_invocation_short_name, "pkill") == 0) {
+        if (strcmp (argv[0], "pkill") == 0) {
 		int sig;
 		prog_mode = PKILL;
 		sig = signal_option(&argc, argv);
@@ -996,9 +998,12 @@
 
 inline static int execute_kill(pid_t pid, int sig_num)
 {
+    // just avoid sigqueue rout for Android
+#if 0
     if (use_sigqueue)
         return sigqueue(pid, sig_num, sigval);
     else
+#endif
         return kill(pid, sig_num);
 }
 
