On-device build support.
diff -uNr src/src/mktokens mod_511_src/src/mktokens > mktokens.patch
--- src/src/mktokens	2021-05-11 17:43:29.000000000 +0530
+++ mod_511_src/src/mktokens	2021-05-11 18:39:23.200300670 +0530
@@ -39,7 +39,7 @@
 
 : "${TMPDIR:=/tmp}"
 
-cat > "${TMPDIR}"/ka$$ <<\!
+cat > ${TMPDIR:-/tmp}/ka$$ <<\!
 TEOF	1	end of file
 TNL	0	newline
 TSEMI	0	";"
@@ -70,28 +70,28 @@
 TBEGIN	0	"{"
 TEND	1	"}"
 !
-nl=`wc -l "${TMPDIR}"/ka$$`
+nl=`wc -l ${TMPDIR:-/tmp}/ka$$`
 exec > token.h
-awk '{print "#define " $1 " " NR-1}' "${TMPDIR}"/ka$$
+awk '{print "#define " $1 " " NR-1}' ${TMPDIR:-/tmp}/ka$$
 
 exec > token_vars.h
 
 echo '
 /* Array indicating which tokens mark the end of a list */
 static const char tokendlist[] = {'
-awk '{print "\t" $2 ","}' "${TMPDIR}"/ka$$
+awk '{print "\t" $2 ","}' ${TMPDIR:-/tmp}/ka$$
 echo '};
 
 static const char *const tokname[] = {'
 sed -e 's/"/\\"/g' \
     -e 's/[^	 ]*[	 ][	 ]*[^	 ]*[	 ][	 ]*\(.*\)/	"\1",/' \
-    "${TMPDIR}"/ka$$
+    ${TMPDIR:-/tmp}/ka$$
 echo '};
 '
-sed 's/"//g' "${TMPDIR}"/ka$$ | awk '
-/TNOT/{print "#define KWDOFFSET " NR-1; print ""; 
+sed 's/"//g' ${TMPDIR:-/tmp}/ka$$ | awk '
+/TNOT/{print "#define KWDOFFSET " NR-1; print "";
       print "static const char *const parsekwd[] = {"}
 /TNOT/,/neverfound/{if (last) print "	\"" last "\","; last = $3}
 END{print "	\"" last "\"\n};"}'
 
-rm "${TMPDIR}"/ka$$
+rm ${TMPDIR:-/tmp}/ka$$
