--- src/src/var.c	2020-06-03 08:16:56.000000000 +0530
+++ mod_511_src/src/var.c	2021-05-13 12:26:19.557083190 +0530
@@ -74,7 +74,7 @@
 MKINIT struct localvar_list *localvar_stack;
 
 const char defpathvar[] =
-	"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin";
+        "PATH=@TERMUX_PREFIX@/bin:@TERMUX_PREFIX@/bin/applets";
 char defifsvar[] = "IFS= \t\n";
 MKINIT char defoptindvar[] = "OPTIND=1";
 
