diff -uNr ./include/llvm/ADT/Triple.h ./mod_include/llvm/ADT/Triple.h > include-llvm-ADT-Triple.h.patch
--- ./include/llvm/ADT/Triple.h	2020-07-07 21:51:37.000000000 +0530
+++ ./mod_include/llvm/ADT/Triple.h	2021-05-03 22:20:00.354303127 +0530
@@ -667,6 +667,10 @@
     unsigned Env[3];
     getEnvironmentVersion(Env[0], Env[1], Env[2]);
 
+	// If not specified, set a default Android API.
+    if (Env[0] == 0)
+      Env[0] = __ANDROID_API__;
+
     // 64-bit targets did not exist before API level 21 (Lollipop).
     if (isArch64Bit() && Env[0] < 21)
       Env[0] = 21;
